### PR TITLE
Bug fixing v2

### PR DESCRIPTION
**Pull request by sakinaboriwala**
### Why?
- Overflow needs to be handled

### This PR resolves the issues by:

- Handle overflow by making the row scrollable

### CODE REVIEW

File: lib/screens/auth.dart [hunk 1]
```diff
@@ -118,11 +118,10 @@ class _AuthenticationState extends State<Authentication>
                     }));
                   },
                   child: Text(
-                    'FORGET YOUR PASSWORD?',
+                    'FORG0T YOUR PASSWORD?',
```
Comment (gopalshimpi) [inline lib/screens/auth.dart]: Typo "Forgot"

**Review by gopalshimpi — CHANGES_REQUESTED** [review]
Please go through for typo fixes

File: lib/screens/auth.dart (hunk 1 above)
Comment (sakinaboriwala) [reply lib/screens/auth.dart]: Okay

**Review by gopalshimpi — APPROVED** [review]
LGTM 👍

**Pull request merged**